README.md

```diff
@@ -78,6 +78,8 @@ List of templates in the theme:
 * User profile page
 * Service list page (optional, service catalog specific)
 * Service page (optional, service catalog specific)
+* Approval request list page
+* Approval request page
```

Comment:
We should mention `(optional, approvals specific)` as we did for Service Catalog. Some customer may not have the Approvals feature and omit these templates in their theme